usdigitalresponse/usdr-gost
# Support archiving subrecipients

### Ticket #1675
## Description

Implements support for "archiving" subrecipients.

An "archived" subrecipient will be excluded from treasury export generation.  For all other purposes, an "archived" subrecipient still behaves identically.  It is still displayed in the UI, and can still be edited, and will still be considered for uniqueness constraints for future uploads.

## Screenshots / Demo Video

<img width="1390" alt="image" src="https://github.com/usdigitalresponse/usdr-gost/assets/11449340/f8262f37-6b98-4f96-ac27-262f206598f0">

<img width="1368" alt="image" src="https://github.com/usdigitalresponse/usdr-gost/assets/11449340/74048f4c-3636-474b-9e48-d6c3383cf030">


## Testing

- `yarn db:migrate`
- `yarn start`
- If not already present, upload a workbook that includes at least one subrecipient
- Trigger a treasury export, _verify that subrecipient is present_
- Navigate to "Subrecipients" tab
- Click "archive", _verify that subrecipient is updated in this view_
- Trigger a treasury export, _verify that subrecipient is no longer present_

### Automated and Unit Tests

### Manual tests for Reviewer

## Checklist
